website/content/docs/nia/configuration.mdx

```diff
@@ -97,7 +97,8 @@ service {
 - `id` - (string) ID identifies the service for Consul-Terraform-Sync. This is used to explicitly identify the service config for a task to use. If no ID is provided, the service is identified by the service name within a [task definition](#task).
 - `name` - (string: required) The Consul logical name of the service (required).
 - `namespace` <EnterpriseAlert inline /> - (string: "default") The namespace of the service. If not provided, the namespace will be inferred from the Consul-Terraform-Sync ACL token, or default to the `default` namespace.
-- `tag` - (string) Tag is used to filter nodes based on the tag for the service.
+- `tag` - (string) **Deprecated** - Use `filter` with the `Service.Tags` selector instead. This field will be removed in a future version of Consul-Terraform-Sync. Tag is used to filter nodes based on the tag for the service.
```

Comment:
I was looking for [consul config examples of deprecated fields](https://www.consul.io/docs/agent/options#_non_voting_server). I kind of like in the example that it included the version that the field was deprecated. What would you think about something like adding in a version in the bolded deprecated message?

Reply:
Yeah, that seems helpful to have the version! I'll update the format to be more like that consul config.